feat: set English locale as default

Если сменить `defaultLocale: 'ru',` на `en`, то:

* `/docs/intro` отдает английскую страницу
* `/ru/docs/intro` выбивает "Page Not Found"

Т.е. нужно как-то заставить Docusaurus отдавать содержимое папки `/docs` по `/ru/docs` роуту. И он это делает, если запускать через `npm run start -- --locale ru`. А всему виной ограничения:

> Each locale is a distinct standalone single-page application: it is not possible to start the Docusaurus sites in all locales at the same time.
> — https://docusaurus.io/docs/i18n/tutorial#start-your-site

## `i18n.localeConfigs.ru.path`

Установка `i18n.localeConfigs.ru.path` на `/docs` не помогла:

```js
i18n: {
  localeConfigs: {
    ru: {
      path: "/docs"
    }
}
```

## `routeBasePath`

Если установить `routeBasePath` на `"ru/docs"`, то ко всем ссылкам добавляется `ru`:

* `/docs/intro` меняется на `/ru/docs/intro` и отдает статью на английском
* `/ru/docs/intro` — на `/ru/ru/docs/intro` и все равно выбивает "Page Not Found"

## Переместить `/docs` в `/i18n/ru`

Если переместить `/docs` в `/18n/ru/docusaurus-plugin-content-docs/current/` то выбивает:

```
[ERROR] Error: The docs folder does not exist for version "current". A docs folder is expected to be found at docs.
```

## Источники

* [Having the default locale in the URL too? #9918](https://github.com/facebook/docusaurus/discussions/9918)
* [Can i set a baseUrl for defaultLocale #8745](https://github.com/facebook/docusaurus/discussions/8745)

--- a/docusaurus.config.ts
+++ b/docusaurus.config.ts
@@ -26,7 +26,7 @@ const config: Config = {
   // useful metadata like html lang. For example, if your site is Chinese, you
   // may want to replace "en" with "zh-Hans".
   i18n: {
-    defaultLocale: 'ru',
+    defaultLocale: 'en',
     locales: ['ru', 'en'],
     localeConfigs: {
       ru: {

--- a/package.json
+++ b/package.json
@@ -4,7 +4,8 @@
   "private": true,
   "scripts": {
     "docusaurus": "docusaurus",
-    "start": "docusaurus start",
+    "start": "docusaurus start --locale ru",
+    "start:en": "docusaurus start",
     "build": "docusaurus build",
     "swizzle": "docusaurus swizzle",
     "deploy": "docusaurus deploy",

